src/apps/allbridge/api.ts

```diff
@@ -97,12 +98,20 @@ export async function getAllbridgeTokenInfo({
     TOKEN_INFO_RESPONSE_KEY,
   ) as AllbridgeApiResponse
   if (!allbridgeTokensInfoResponse) {
+    const headers: Record<string, string> = {
+      'x-Sdk-Agent': 'AllbridgeCoreSDK/3.21.0', // as in https://github.com/allbridge-io/allbridge-core-js-sdk/blob/5deb0623d6fffcbc7a85dba22b98942c47d4af6c/src/client/core-api/api-client.ts#L48
+    }
+
+    const allBridgeHeader = getConfig().ALLBRIDGE_HEADER
```

Comment:
Could we just specify the API key in the secret and assign it to a static header here? It seems like if something would change, it would be the API key value, and not the header key itself; this also removes room for error in setting up the secret (since nothing would complain about an incorrect header key in the secret itself).

Reply:
Changed in 5b41c91da28f015284c2747f819fd61e45ee799d!